### ISSUE TITLE
Demo code

### ISSUE THREAD

**Issue by fastlater** [hjweide/convnet-for-geometric-matching#2]
Hi,
I can see your results in the readme but do you have a demo code to visually these results? Like 
https://github.com/ignacio-rocco/cnngeometric_pytorch/blob/master/demo.py

I was following the original project and I would like to test your code and provide feedback if I find something that may interest you.

Thanks and advance.

**Comment by hjweide** [comment]
Yes, please see the [warp_images.py](https://github.com/hjweide/convnet-for-geometric-matching/blob/master/warp_images.py#L28) file.  You can specify pairs of filenames for warping.  The results are the three images plotted together as one (like the example images).

Note that I haven't implemented the thin-plate-spline transform yet, so I don't expect my code to reproduce the results from the paper.  If you find any bugs or other problems with my code, please let me know though :)

**Comment by fastlater** [comment]
@hjweide Thank you for your quick reply. I will check the warp_images.py. Actually, I am only interested on the affine results. I would like to reproduce the  affine results using tensorflow and your code in theano is easier to read for me than pytorch. Let me know if you know any github where someone already did it in tensorflow so I can follow that project too.
